apis/v1alpha2/httproute_types.go

```diff
@@ -704,6 +769,31 @@ type HTTPRequestRedirectFilter struct {
 	StatusCode *int `json:"statusCode,omitempty"`
 }
 
+// HTTPURLRewriteFilter defines a filter that modifies a request during
+// forwarding. At most one of these filters may be used on a Route rule. This
+// may not be used on the same Route rule as a HTTPRequestRedirect filter.
```

Comment:
nit - for consistency
```suggestion
// MUST NOT be used on the same Route rule as a HTTPRequestRedirect filter.
```

Reply:
Good catch - will be fixed by #1160

Reply:
Updated, thanks!